src/integrity-checker.js

```diff
@@ -247,6 +251,7 @@ export default class InstallationIntegrityChecker {
       integrityFileMissing: false,
       integrityMatches,
       missingPatterns,
+      artifacts: expected ? expected.artifacts : null,
```

Comment:
Returning artifacts from check seems to break cohesion of the check function.
Makes sense from performance point of view because we already parse integrity file here but I would rather have a separate function that returns integrity file.

Side note: my fault that check command has side effects (reporter logs), I plan to refactor it.